batch/provision.yml

```diff
@@ -10,6 +10,9 @@
     - master_instance_type: m1.medium
     - core_instance_num: 1
     - core_instance_type: m1.medium
+    - tags:
+        Key: 'deployment'
```

Comment:
This is the name of the tag to be created?
In that case, it should have a leading capital letter, `Deployment`.

Reply:
We should still do this, right?